index.html

```diff
@@ -52,7 +52,73 @@ <h3 class="stories">Multi-Post Stories</h3>
+        <a href="#" class="section3-item3"><li>Ruby</li></a>
+      </ul>
+      <a href="#" class="section3__project">See Project</a>
+    </section>
+    <section class="section3">
+      <h4 class="section3-heading">Proffesional Art Printing Data</h4>
+      <p class="section3-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="section3-items">
+        <a href="#" class="section3-item1"><li>HTML</li></a>
+        <a href="#" class="section3-item2"><li>Bootstrap</li></a>
+        <a href="#" class="section3-item3"><li>Ruby</li></a>
+      </ul>
+      <a href="#" class="section3__project">See Project</a>
+    </section>
+    <section class="section3">
+      <h4 class="section3-heading">Proffesional Art Printing Data</h4>
+      <p class="section3-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="section3-items">
+        <a href="#" class="section3-item1"><li>HTML</li></a>
+        <a href="#" class="section3-item2"><li>Bootstrap</li></a>
+        <a href="#" class="section3-item3"><li>Ruby</li></a>
+      </ul>
+      <a href="#" class="section3__project">See Project</a>
+    </section>
+    <section class="section3">
+      <h4 class="section3-heading">Proffesional Art Printing Data</h4>
+      <p class="section3-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="section3-items">
+        <a href="#" class="section3-item1"><li>HTML</li></a>
+        <a href="#" class="section3-item2"><li>Bootstrap</li></a>
+        <a href="#" class="section3-item3"><li>Ruby</li></a>
+      </ul>
+      <a href="#" class="section3__project">See Project</a>
+    </section>
+    <section class="section3">
+      <h4 class="section3-heading">Proffesional Art Printing Data</h4>
+      <p class="section3-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="section3-items">
+        <a href="#" class="section3-item1"><li>HTML</li></a>
+        <a href="#" class="section3-item2"><li>Bootstrap</li></a>
+        <a href="#" class="section3-item3"><li>Ruby</li></a>
+      </ul>
+      <a href="#" class="section3__project">See Project</a>
+    </section>
+    <section class="aboutme-section">
+    <p class="aboutme-heading">About me</p>
+    <p class="aboutme-paragraph">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <a href="#" class="aboutme-button">Get My Resume</a>
+    <img class="vector-line" src="images/Vector4.png" alt="images">
```

Comment:
- [ ] (OPTIONAL) An alternative here would be an `hr` tag which is a straight line.